src/assets/scss/app.scss

```diff
@@ -1,4 +1,5 @@
-@import 'variables';
-@import 'base';
+// Import all component scss files from this directory.
+@import '_*.scss';
```

Comment:
I think you just want to import the components one by one since the order of them could matter.

Reply:
Yes